src/script/components/UserSearchableList.tsx

```diff
@@ -100,45 +99,35 @@ const UserSearchableList: React.FC<UserListProps> = ({
 
   useEffect(() => {
     const connectedUsers = conversationState.connectedUsers();
-    let resultUsers = users.slice();
-    const normalizedQuery = SearchRepository.normalizeQuery(filter);
-    if (normalizedQuery) {
-      const trimmedQuery = filter.trim();
-      const isHandle = trimmedQuery.startsWith('@') && validateHandle(normalizedQuery);
-      const SEARCHABLE_FIELDS = SearchRepository.CONFIG.SEARCHABLE_FIELDS;
-      const properties = isHandle ? [SEARCHABLE_FIELDS.USERNAME] : undefined;
```

Comment:
This logic of determining which fields to query was duplicated. This has been factored to the `SearchRepository`. 
Now the SearchRepository has the logic to know if it's a handle search or regular search and can also act accordingly